conda_smithy/cli.py

```diff
@@ -54,62 +98,87 @@ class Init(Subcommand):
-        meta = MetaData(args.recipe_directory)
-        feedstock_directory = args.feedstock_directory.format(package=argparse.Namespace(name=meta.name()))
-        generate_feedstock_content(feedstock_directory, args.recipe_directory)
+        is_multi = args.multi
+        # check some error conditions
+        if args.recipe_directory and not os.path.isdir(args.recipe_directory):
+            raise IOError("The source recipe directory should be the directory of the "
+                          "conda-recipe you want to build a feedstock for. Got {}".format(
+                args.recipe_directory))
+
+        # Get some information about the source recipe.
+        if args.recipe_directory:
+            meta = MetaData(args.recipe_directory)
+        else:
+            meta = None
+
+        if is_multi:
+            feedstock_directory = args.feedstock_directory.format(package=argparse.Namespace(
+                name="multi-package"))
+            msg = 'Initial commit of the multi packages feedstock.'.format(meta.name())
+        else:
+            feedstock_directory = args.feedstock_directory.format(package=argparse.Namespace(name=meta.name()))
+            msg = 'Initial commit of the {} feedstock.'.format(meta.name())
+
+        generate_feedstock_content(feedstock_directory, args.recipe_directory, is_multi, meta)
         if not args.no_git_repo:
-            create_git_repo(feedstock_directory, meta)
+            create_git_repo(feedstock_directory, msg)
 
+        print("\nRepository created, please edit conda-forge.yml to configure the upload channels\n"
+              "and afterwards call 'conda smithy register-github ...'")
 
-class GithubCreate(Subcommand):
-    subcommand = 'github-create'
+class RegisterGithub(Subcommand):
+    subcommand = 'register-github'
     def __init__(self, parser):
-        #  conda-smithy github-create ./ --organization=conda-forge
-        subcommand_parser = Subcommand.__init__(self, parser, "Create a github repo for a feedstock")
-        subcommand_parser.add_argument("feedstock_directory")
-        group = subcommand_parser.add_mutually_exclusive_group()
-        group.add_argument("--user")
-        group.add_argument("--organization", default="conda-forge")
-        subcommand_parser.add_argument("--remote-name", default="upstream",
+        #  conda-smithy register-github ./ --organization=conda-forge
+        super(RegisterGithub, self).__init__(parser, "Register a repo for a feedstock at github.")
+        scp = self.subcommand_parser
+        scp.add_argument("feedstock_directory",
+                         help="The directory of the feedstock git repository.")
+        group = scp.add_mutually_exclusive_group()
+        group.add_argument("--user", help="github username under which to register this repo")
+        group.add_argument("--organization", default="conda-forge",
```

Comment:
IMO this default should be removed and the user forced to add a config option. I think it will happen too often that a user simply does not set this switch and gets an error that they have no rights to add a repo to the conda-forge org...

Or maybe a conda-smith config file which specifies that?